packages/seed/src/cli.ts

```diff
@@ -42,7 +42,7 @@ export async function tryRunCli(): Promise<void> {
 
     addTestCommand(cli);
     addRunCommand(cli);
-    addRegisterCommands(cli);
+    addBuildCommand(cli), addRegisterCommands(cli);
```

Comment:
nit: lets move to a new line

Reply:
Embarrassing typo (`,` instead of `;` that got auto-fmt'd to same line). Fixed in next commit.